channeld/channeld.c

```diff
@@ -2016,20 +2017,22 @@ static struct commitsig_info *handle_peer_commit_sig(struct peer *peer,
 		outpoint = peer->splice_state->inflights[commit_index - 1]->outpoint;
 		funding_sats = peer->splice_state->inflights[commit_index - 1]->amnt;
 
-		if (!cs_tlv || !cs_tlv->splice_info)
+		if (cs_tlv && cs_tlv->splice_info
+		    && cs_tlv->splice_info->batch_size == 1)
 			peer_failed_err(peer->pps, &peer->channel_id,
-					"Must set funding_txid for each"
-					" extra commitment_signed message.");
+					"batch_size can never be 1");
 
-		status_info("handle_peer_commit_sig for inflight outpoint %s", fmt_bitcoin_txid(tmpctx, &peer->splice_state->inflights[commit_index - 1]->outpoint.txid));
-		status_info("handle_peer_commit_sig cs_tlv->splice_info->funding_txid %s", fmt_bitcoin_txid(tmpctx, &cs_tlv->splice_info->funding_txid));
+		status_debug("handle_peer_commit_sig for inflight outpoint %s",
+			     fmt_bitcoin_txid(tmpctx, &outpoint.txid));
+		status_debug("handle_peer_commit_sig cs_tlv->splice_info->funding_txid %s",
```

Comment:
Should be within an `if (cs_tlv && cs_tlv->splice_info)` check.

Reply:
This happens on line 2360